tox.ini

```diff
@@ -31,6 +31,19 @@ commands =
     python --version
     pytest -p no:warnings --cov=baybe --durations=5 {posargs}
 
+[testenv:singlepointprecision,singlepointprecision-py{310,311,312}]
+description = Run specific tests with single point precision
+extras = test
+passenv =
+    CI
+setenv =
+    BAYBE_NUMPY_USE_SINGLE_PRECISION = true
+    BAYBE_TORCH_USE_SINGLE_PRECISION = true
+    BAYBE_TEST_ENV = SINGLEPOINT
+commands =
+    python --version
+    pytest -p no:warnings tests/constraints/
```

Comment:
why is this just limited to constraints? I woudl expect a singlepoint precision test of the whole suite

Reply:
I was unsure if we actually want to run everything also in singlepoint precision, specifically since some tests will probably fail (we already have problems with numerical stability). Hence, I just added the tests that were failing previously, we can however also add more/all tests, but then they should probably be allowed to fail.

Reply:
Have changed this such that everything is being tested. However, not resolving this in case of @AdrianSosic wanting to give his opinion here resp. to already have a thread open if this yields problems due to stability.

Reply:
I think the `tox` command itself should not be limited to a subset but rather test the whole suite. The reason is that you want to be able to flexibly use this programmatically and we can always filter down by passing additional arguments to the cli.

However, I rather wonder how we want to do stuff in CI. That is, do we want to add yet another entire test run? Is perhaps a bit of an overkill. Right now, we have one core test and two full tests. Maybe we should simply change one of the full tests to singlepoint?

Reply:
The problem that I see here is that single point test for the whole test suite might fail quite regularly due to numerical instabilities, hence I am not sure if we should replace one of our tests.

Reply:
Since all of these tests have been removed for now, I resolve this.